variant-list.js

```diff
@@ -959,6 +996,31 @@ let variant_list_data = [
         min_players_per_team:1,
         max_players_per_team:1
     },
+    {
+        name: "Oldsitego",
+        credit: "ad",
+        description: "A variant of lockout (or blackout) in which all the objectives are Old Site related",
+        external_links: [{
+            name:"Generator",
+            link:"download-files/oldsitego.json"
```

Comment:
This should be file, not link, links are for external websites, file for files in the depo